<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBox</class>
 <widget class="QGroupBox" name="GroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Map</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGraphicsView" name="map_view"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="button_layout">
     <item>
      <widget class="QPushButton" name="zoom_in_button">
       <property name="text">
        <string>Zoom In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoom_out_button">
       <property name="text">
        <string>Zoom Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoom_reset_button">
       <property name="text">
        <string>Reset Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="value_spin_box"/>
     </item>
     <item>
      <widget class="QPushButton" name="add_value_button">
       <property name="text">
        <string>Add Value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
